pkg/eventemitter/eventemitter.go

```diff
@@ -324,6 +331,14 @@ func (e *EventEmitter) emitEventByHandler(eventData eventdata.EventData) {
 
 	if eventData.HandlerKey == "" {
 		for key, handler := range e.eventHandlersCache {
+			// Filter Event
+			identifierKey := getPrefixIdentifierFromKey(key)
+
+			if e.eventFilterCache[identifierKey] != nil && !e.eventFilterCache[identifierKey].FilterEvent(eventData.CloudEventType) {
```

Comment:
Should we introduce a RLock before accessing to this cache? (with the propel Lock when it's being modified)